framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -770,35 +805,28 @@ public AnnotationFormatter getAnnotationFormatter() {
     // Factories for annotated types that account for implicit qualifiers
     // **********************************************************************
 
-    /** Should results be cached? Disable for better debugging. */
-    protected static boolean SHOULD_CACHE = true;
-    public boolean shouldCache = SHOULD_CACHE;
-
-    /** Should the cached result be used, or should it be freshly computed? */
-    protected static boolean SHOULD_READ_CACHE = true;
-    public boolean shouldReadCache = SHOULD_READ_CACHE;
-
-    /** Size of LRU cache. */
-    private final static int CACHE_SIZE = 300;
-
-    /** Mapping from a Tree to its annotated type; implicits have been applied. */
-    private final Map<Tree, AnnotatedTypeMirror> treeCache = CollectionUtils.createLRUCache(CACHE_SIZE);
-
-    /** Mapping from a Tree to its annotated type; before implicits are applied,
-     * just what the programmer wrote. */
-    protected final Map<Tree, AnnotatedTypeMirror> fromTreeCache = CollectionUtils.createLRUCache(CACHE_SIZE);
-
-    /** Mapping from an Element to its annotated type; before implicits are applied,
-     * just what the programmer wrote. */
-    private final Map<Element, AnnotatedTypeMirror> elementCache = CollectionUtils.createLRUCache(CACHE_SIZE);
-
-    /** Mapping from an Element to the source Tree of the declaration. */
-    private final Map<Element, Tree> elementToTreeCache  = CollectionUtils.createLRUCache(CACHE_SIZE);
-
     /** Mapping from a Tree to its TreePath **/
     private final TreePathCacher treePathCache = new TreePathCacher();
 
     /**
+     * Returns the int supplied to the checker via atfCacheSize or
```

Comment:
Instead of "via atfCacheSize" I would make clear that you mean an option by that name, not a field or a parameter.